src/cart/src/gurt/fault_inject.c

```diff
@@ -627,6 +619,14 @@ d_fi_initialized()
 	return d_fi_gdata.dfg_inited == 1;
 }
 
+bool
+d_fault_inject_yml_enable(void)
```

Comment:
The name of this function implies 2 things:
- that its yml based only
- that is enables fault injection

However fault injection can also be enabled at runtime (see crt_hdlr_ctl_fi_toggle(crt_rpc_t *rpc_req))
And also this function does not enable fault injection, it simply checks if it is enabled.

as such this should probably be better named as
d_fault_inject_is_enabled()  or something like that

Reply:
ok.